khronos_api/build.rs

```diff
@@ -0,0 +1,25 @@
+use std::env;
+use std::fs::File;
+use std::path::*;
+use std::io::Write;
+
+fn main() {
+    let dest = env::var("OUT_DIR").unwrap();
+    let mut file = File::create(&Path::new(&dest).join("webgl_exts.rs")).unwrap();
+
+    let root = env::current_dir().unwrap().join("api_webgl/extensions");
+
+    writeln!(file, "&[").unwrap();
+    for entry in root.read_dir().unwrap() {
+        let entry = entry.unwrap();
+        let path = entry.path();
+        let ext_name = path.file_name().unwrap().to_str().unwrap();
+        if path.is_dir() && ext_name != "template" {
+            let ext_path = path.join("extension.xml");
+            if ext_path.is_file() {
+                writeln!(file, "&*include_bytes!({:?}),", ext_path.to_str().unwrap()).unwrap();
+            }
+        }
+    }
+    writeln!(file, "]").unwrap();
```

Comment:
Why is this needed? Perhaps a small comment could help make it clearer, along with pointing to the `WEBGL_EXT_XML` constant, where this will be used?

Reply:
Oh, saw the comment in https://github.com/brendanzab/gl-rs/pull/447#issuecomment-363089825 now - I think it still could use some added info though.

Reply:
Yeah, I'll add some comments explaining what's going on here - is there any other information you think would be worth adding?

Reply:
Nope, that should be fine! I think folks should get the gist of it from the code once they are in the right frame of mind.